### ISSUE TITLE
10 Graphs to Preview the Start of the German Bundesliga Season

### ISSUE THREAD

**Issue by barjacks** [jsoma/playfair-projects#72]
**1 Graph of total market value of the Bundesliga**

**5 Graphs on players **
- Graphs of new vs. old players. (1) Games, (2) Age, (3) Goals, (4) Assists, (5) Height.

** 2 Graphs on Shoes **
- Graph by number of games
- Graph of number of goals scored

**2 Graphs on Most Active Agents **
- Graph by amount of players
- Graph by amount of player value

Data source is a scraper I set up using Beautiful Soup, that scraps this footballing site [transfermarkt.com](http://www.transfermarkt.com) and the respective player's profile pages:  

## Story issue checklist

My pitch was (use the number): #10 

- [x] My pitch has been approved (see `PITCHING.md`)
- [x] My story issue links to my pitch issue
- [x] I link to my finalized (ish) data source(s)
- [x] I've included a brief summary of my story
- [x] I've included some possible headlines or findings
- [x] I've included some links or images as inspiration (if you have any)
- [x] I've included an update of my visualization/story in a comment
- [x] I have received two comments of peer feedback
- [ ] I have received editorial feedback

[Bundesliga_Transfer_Bilanz.pdf](https://github.com/jsoma/playfair-projects/files/387235/Bundesliga_Transfer_Bilanz.pdf)

**Comment by jsoma** [comment]
You'll want to paste in your sample as a `png` instead of a `pdf` so it'll be easier for folks to view it without downloading/opening (and that will be the final format for publication, too). I also went ahead and checked some of your boxes for you, hope you don't mind!

**Comment by barjacks** [comment]
Thanks @jsoma. I've cleaned the main graph a little more and am uploading it as png. Now I need some colleagues to comment on it. And thanks for check boxing. I completely forgot. Editorial feedback is from you again?

![bundesliga_transfer_bilanz](https://cloud.githubusercontent.com/assets/1587242/17224329/801e45a2-54ce-11e6-85b9-4aaa1ad3be7b.png)

**Comment by barjacks** [comment]
Thanks @jsoma. I've cleaned the main graph a little more and am uploading it as png. (it's also in English now, better for class to read I guess, if I want their comments. And thanks for check boxing. I completely forgot. Editorial feedback is from you again?
![bundesliga_transfer_bilanz](https://cloud.githubusercontent.com/assets/1587242/17224481/0d41206c-54cf-11e6-9751-8e1266e028f2.png)

**Comment by barjacks** [comment]
Here are some more graph examples. Not yet as clean as the example above and with German head lines:

Comparing age differences of new players vs old players
![alter](https://cloud.githubusercontent.com/assets/1587242/17224789/5dda7d92-54d0-11e6-963f-4ad71c1ecb10.png)

Size differences
![groesse](https://cloud.githubusercontent.com/assets/1587242/17224800/70d128ec-54d0-11e6-9032-7598424708e4.png)

Whichs shoes scored the most goals. (when I scraped the site I also scraped the shoes of each players, where available)
![schuhe](https://cloud.githubusercontent.com/assets/1587242/17224829/89005758-54d0-11e6-90a4-d7c47c55272c.png)

**Comment by barjacks** [comment]
Improvement of the shoe graph, I think so anyway :-).

![schuhe](https://cloud.githubusercontent.com/assets/1587242/17226087/ae730242-54d5-11e6-913c-264d747278ae.png)

**Comment by barjacks** [comment]
Improvement of the shoe graph, I think so anyway :-).
![schuhe](https://cloud.githubusercontent.com/assets/1587242/17226111/d317658e-54d5-11e6-9aa3-4d0f716c870a.png)

**Comment by barjacks** [comment]
And here's the scraping, which is essentially the data source of course. Any comments on this very welcome as well:
https://github.com/barjacks/fussball

**Comment by gcgruen** [comment]
Hey @barjacks -- I liked your graphic a lot, especially given the fact that I don't know anything about football, apart from my heart being #schwatzgelb ;)

I like your last one most, because of the little shoe icons -- that's so cool to not have boring names but actually see them!
With the xaxis-label I was not so sure about where you've put it, because just randomly in the middle of the chart, it appears to me more like an annotation to a single bar than like the xaxis label.

With all other ones, you'd need to add your Twitter handle and the data source. And I would only have tiny suggestions for the case you'd like to spend more time on this:
Graphic 1: Being not into soccer too much, I'm not too sure how I have to read the graphic to see how the headline is derived from it. I can't really say how to improve it, bc I know to little -- maybe stack two of the three bars to see that one is significantly higher than the other?

Graphic 2: You'd need to change the axis-Label from "Spielminuten" to age, I guess from the headline? If the axis is age -- what age equals the 0 value then?

Graphic 3: Would maybe add y-axis Label "cm" in case it is for an international audience and people dare to ignore to read the headline ;)

Hope this helps, feel free to let me know if sth needs clarifying.

**Comment by shuyao1201** [comment]
Hi Barney:

I love the first and the last one. It's legible to to put when the profits are negative

**Comment by shuyao1201** [comment]
Hi Barney:

Though I don't know much abhout football, I love the first graph and the last one.  it's legible to divide profits barh into two left part and right part from the middle. Also the white background highlight the data we need to focus on. It would be better if the grid dash-line is bit more clear or bolder. 

As to the last one, the summary could be the subtitle under the title, upper the first bar. Inspired by Soma's comment on mine annotations #39 ，put all the shoes' name on the top or under the shoe. Also, a thinner bar.

**Comment by thisss** [comment]
Hi Barney, 

wow, that was a lot of work. I guess, some of the real football fans will love to look at these facts. Just some short remarks: 
- You might want to change the order of the legend of the first graph so that it corresponds with the order of the bars on the left. 
- Maybe you could change the Augstburg title to: "Augsburg setzt auf grosse Spieler", "Augsburg zeigt Grösse" or "Augsburg kauft gross ein".   
- The shoes are great. But they need to be bigger. I want to really see the best scoring shoes! By the way: Very good idea -- surprising and funny.

**Comment by barjacks** [comment]
Thanks for all of your comments. 

@gcgruen you're right. I forgot the data source in all of my graphs in the middle. I have out them in now, and colored the positive valous. I'm not sure how to improve the readibility of the first graph. The clubs are sorted by the red bar, how much cash they actually earned in the transfer period. 

![alter](https://cloud.githubusercontent.com/assets/1587242/17271157/c724634e-5642-11e6-8c7b-94c9a2c72908.png)
![alter](https://cloud.githubusercontent.com/assets/1587242/17271158/cdb806fc-5642-11e6-8357-cc1048f9fb2b.png)

**Comment by barjacks** [comment]
Thanks for all of your comments.

@gcgruen you're right. I forgot the data source in all of my graphs in the middle. I have out them in now, and colored the positive valous. I'm not sure how to improve the readibility of the first graph. The clubs are sorted by the red bar, how much cash they actually earned in the transfer period.
![erfahrung](https://cloud.githubusercontent.com/assets/1587242/17271170/fba3ba8e-5642-11e6-9b14-e9f17550f1b9.png)
![alter](https://cloud.githubusercontent.com/assets/1587242/17271172/fe757c02-5642-11e6-86a9-d6417d85895b.png)

**Comment by barjacks** [comment]
Thanks a lot @shuyao1201, I played around with making the grid lines more clear, but they then just seem to dominate the graphic. So I just left it. You are right about the subtitle of the third graph. I changed that. And also incorporated @thisss comment on the size of the shoes. Think it works much better like this. Thanks a lot to both of you! 
![schuhe](https://cloud.githubusercontent.com/assets/1587242/17271188/54375e44-5643-11e6-9ab3-8ff82de57780.png)
